.gitignore

```diff
@@ -27,8 +27,8 @@ coverage.txt
 
 EXTRACTED_CHANGELOG.md
 
-cue.mod/pkg
-cue.mod/usr
+cue.mod/gen
+/pkg/model/api/v1/**/*_go_gen.cue
```

Comment:
I think we are saving these files instead of ignoring them, then it will ease the DAC setup because you won't need to import the go definition to generate the cue definition.

Reply:
Ok yeah I suppose this is a valid workaround, but ideally we should rely on this CUE feature + not commit files generated in a deterministic way.

Reply:
After discussion, agreed to commit these files & do some operations to have them in a dedicated folder